content/docs/other-guides/freq-ask-questions.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Frequently asked questions"
+description = ""
+weight = 110
++++
+
+
+### Does Kubeflow Support Helm?
```

Comment:
I think the answer is slightly more complicated. 
Kubeflow only provides kustomize packages for KF applications. So deploying KF isn't supported except via kustomize.

Its true that once you deploy KF you can express K8s resources using whatever tool you want.

So the wording of the question is a bit ambiguous.


Lets not mention ksonnet since ksonnet has reached end of life.

Reply:
Looking at the original issue: https://github.com/kubeflow/website/issues/450#issue-403876948

My original comment on that thread is a bit dated. At that time we were providing ksonnet packages for TFJob and TFServing but that is no longer the case.

To make this question less ambiguous I might suggest phrasing it as "Is deploying Kubeflow with Helm supported?"
And the answer should be no.

Reply:
@jlewi I have updated page please take a look. Thanks